app/graphql/types/integrations/netsuite.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Types
+  module Integrations
+    class Netsuite < Types::BaseObject
+      graphql_name 'NetsuiteIntegration'
+
+      field :code, String, null: false
+      field :id, ID, null: false
+      field :name, String, null: false
+      field :account_id, String, null: true
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `account_id` should appear before `name`.